cli/prerender.js

```diff
@@ -0,0 +1,30 @@
+// Load zone.js for the server.
+require('zone.js/dist/zone-node');
+const fs = require('fs');
+
+// Import renderModuleFactory from @angular/platform-server.
+const { renderModuleFactory } = require('@angular/platform-server');
+
+// Import module map for lazy loading
+const { provideModuleMap } = require('@nguniversal/module-map-ngfactory-loader');
+
+const files = fs.readdirSync(`${process.cwd()}/dist/dist-server`);
+const mainFiles = files.filter(file => file.startsWith('main'));
+const hash = mainFiles[0].split('.')[1];
+
+// Import the AOT compiled factory for your AppServerModule.
+// This import will change with the hash of your built server bundle.
+const { AppServerModuleNgFactory, LAZY_MODULE_MAP } = require(`./dist/dist-server/main.${hash}.bundle`);
+
+// Load the index.html file.
+const index = require('fs').readFileSync('./dist/index.html', 'utf8');
+
+// Render to HTML and log it to the console.
```

Comment:
Is this comment correct now? 

You're not doing `console.log(html)`, it's ` fs.writeFileSync('./dist/index.html', html)`

Maybe you can change the comment to 
> writes data to `./dist/index.html` file, replacing the file if it already exists.

Reply:
Thanks, nice spotting, fixed